### PR TITLE
Add material-design-lite

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by dougalsutherland** [comment]
See discussion at https://github.com/conda-forge/staged-recipes/pull/4044#issuecomment-332019807.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge.

File: recipes/material-design-lite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
```
Comment (isuruf) [inline recipes/material-design-lite/meta.yaml]: Can you remove this line?

Comment (sannykr) [reply recipes/material-design-lite/meta.yaml]: Thanks for the review. I have removed that line.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Please let me know if you have more suggestions.

**Comment by scopatz** [comment]
Thanks @sannykr!

**Pull request merged**